apps/spruce/src/pages/version/TestAnalysis/index.tsx

```diff
@@ -124,18 +128,11 @@ const TestAnalysis: React.FC<TestAnalysisProps> = ({ versionId }) => {
           </Body>
           <FilterContainer>
             <div>
-              <LabelWrapper>
-                <Label
-                  htmlFor="test-failure-search"
-                  id="test-failure-search-label"
-                >
-                  Search Test Failures
-                </Label>
-              </LabelWrapper>
               <TextInputWithValidation
```

Comment:
Sadly performance on this input is super laggy 🙁 I observed this on version `mongodb_mongo_master_ea79e18886a3237740b9a16be5e400dd12edf277` on prod data

Reply:
hmm I've tried several optimizations. While this performs well on prod when Strict mode is turned off; the poor performance on the local dev server does indicate there is probably an underlying performance problem using live search. I've updated this to now require manual submission like the rest of the inputs on the page. (The user will need to press enter to submit the search) Thanks for calling this out!